openassessment/xblock/submission_mixin.py

```diff
@@ -806,7 +820,6 @@ def submission_path_and_context(self):
             context['team_file_urls'] = self.file_manager.team_file_descriptors(
                 team_id=team_id_for_current_submission
             )
-        if self.file_upload_type == 'custom':
```

Comment:
Why are we removing this check?

Reply:
We want to always surface the `white_listed_file_types` list when file uploads are enabled, not just for `custom` config but for presets as well.